<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1657</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>393</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>393</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_current_date">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="text">
         <string>Текущая дата</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Дата</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Время</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_nameEvent"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_noteEvent"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addEvent">
          <property name="text">
           <string>Добавить событие</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="table_events">
        <column>
         <property name="text">
          <string>Сделано</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Время</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Заметка</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Удалить</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_reminder">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Напоминание о предстоящих событиях:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_reminderEvents">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_HabitTracker">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Трекер привычек</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_habitName">
          <property name="text">
           <string>Название привычки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_habitName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_repetitionRate">
          <property name="text">
           <string>Повторение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_repetitionRate"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addHabit">
          <property name="text">
           <string>Добавить привычку</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="table_habitTracker">
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Повторение</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Последнее время</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Последняя дата</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Статус</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Удалить</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1657</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
